pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -69,7 +70,13 @@ def on_trainer_init(
         # it is important that these are the last callbacks to run
         self.trainer.callbacks = self._reorder_callbacks(self.trainer.callbacks)
 
-    def configure_checkpoint_callbacks(self, checkpoint_callback: Union[ModelCheckpoint, bool]):
+    def _configure_checkpoint_callbacks(self, checkpoint_callback: bool) -> None:
+        if not isinstance(checkpoint_callback, bool):
+            error_msg = "Invalid type provided for checkpoint_callback: "
+            f"Expected bool but received {type(checkpoint_callback)}."
```

Comment:
```suggestion
            error_msg = (
                "Invalid type provided for checkpoint_callback:"
                f" Expected bool but received {type(checkpoint_callback)}."
            )
```
needs to be surrounded by parentheses, otherwise the second line will not be part of the error_msg